packages/site/src/data.js

```diff
@@ -0,0 +1,15 @@
+export const brands = [
```

Comment:
Just curious, in the future are we gonna try to pull this dynamically from somewhere, or will we have to maintain this list to keep it up to date?

Reply:
Yep, definitely we should pull it dynamically - I'll make a ticker for it so we don't forget